packages/extension-chakra-storefront/CHANGELOG.md

```diff
@@ -4,5 +4,6 @@
 - Send PWA Kit events to Data Cloud [#318] (https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2229)
 - Fix dependencies vulnerabilities [#2338](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2338)
 - Fix accessibility issues [#2375](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2375)
+- Fix data-fetching regression with SSR rendering [#2785](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2785)
```

Comment:
I'm wondering if we want to be a little be more clear on this for discoverability purposes.

We can either say what you have but mention + "due to chakra v3 upgrade".

So we can say something like "Ensure client-side only API's are called on the server"

I'm also ok with leaving it as is.